packages/xo-web/src/common/rich-text.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { connectStore } from 'utils'
+import { createGetObjectsOfType } from 'selectors'
+import PropTypes from 'prop-types'
+import decorate from 'apply-decorators'
+import { injectState, provideState } from 'reaclette'
+import Copiable from 'copiable'
+import { flatMap } from 'lodash'
+import Link from 'link'
+import store from 'store'
+
+const RichText = decorate([
```

Comment:
We do not understand by the name of the component, it will convert the IDs into clickable links.

Reply:
The name is generic, in case in the future we need to use the component for more operations than converting only IDs into links.